.travis.yml

```diff
@@ -16,6 +18,8 @@ env:
 before_script:
   - ./dev/bots/travis_setup.sh
 script:
+  - export BRANCH=$(if [ "$TRAVIS_PULL_REQUEST" == "false" ]; then echo $TRAVIS_BRANCH; else echo $TRAVIS_PULL_REQUEST_BRANCH; fi)
```

Comment:
Can these lines be moved to the main script? Or are the variables needed not available there?